src/locale/misc/smartSearch/en.yml

```diff
@@ -0,0 +1,55 @@
+headers:
+    current: Current target group
+    add: Add a new filter
+    examples: Examples
+buttonLabels:
+    cancel: Cancel
+    save: Save
+    add: Add filter to target group
+    edit: Edit filter
```

Comment:
I think this should be "Save" or "Update" instead:

![image](https://user-images.githubusercontent.com/550212/126672811-b13a2723-c361-4df7-ae8a-0bf2628d09e5.png)

Reply:
Also, less take this opportunity to ban the word "filter" and use "selection" or "group" something like that instead. 😊